dependabot.yml

```
@@ -0,0 +1,17 @@
+    schedule:
+      interval: "weekly"
+    target-branch: "main"
+    dependency-type: "all"
+
+  # For the `1.0.0` release branch
+  - package-ecosystem: "yarn"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    target-branch: "1.0.0"
+    dependency-type: "production"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)